packages/test/test-end-to-end-tests/src/test/loadModes.spec.ts

```diff
@@ -273,15 +273,16 @@ describeCompat("LoadModes", "NoCompat", (getTestObjectProvider, apis: CompatApis
 			dataObject1.increment();
 		}
 		await loaderContainerTracker.ensureSynchronized(container1);
-		const result = summarizer.summarizeOnDemand({ reason: "test" });
-		const submitResult = await result.receivedSummaryAckOrNack;
-		assert.ok(submitResult);
+		const result = await summarizeNow(summarizer);
 
 		// Record sequence number we want to pause at, and the expected value at that sequence number
 		const sequenceNumber = container1.deltaManager.lastSequenceNumber;
 		const expectedValue = dataObject1.value;
 
-		const headers: IRequestHeader = {};
+		const headers: IRequestHeader = {
+			// Force the container to load from the latest created summary instead of using the cached version. Latest snapshot is in cache is updated async so could cause test flakiness.
+			[LoaderHeader.version]: result.summaryVersion,
```

Comment:
This probably only works for local server and tinylicious, I'm pretty sure it doesn't work for ODSP as they only keep one version of the snapshot (and you are going to run into cache), and I'm not sure about AFR.

Reply:
@tyler-cai-microsoft Are you recommending to not go ahead with this change? How do I make it work of ODSP? Should I keep it as it was before?

Reply:
It will work for ODSP as well. The reason this is needed is to prevent hitting the cache in ODSP.

Reply:
The case where it won't work is if you are requesting a summary older than the last one which isn't the case here.